src/styles/blocs/nav.scss

```diff
@@ -37,5 +22,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color $transition-duration linear;
```

Comment:
This is optional thing, but may be you can play with type of timing functions. Linear is good, but with 300ms transition shows this effect too weak. You can try ease-in or ease-in-out. They are much more visible for such short duration.